### PR TITLE
docs(changelog) add missing entries

### PR DESCRIPTION
**Pull request by mayocream**
### Summary

Add missing changelog for:
- https://github.com/Kong/kong/pull/8255
- https://github.com/Kong/kong/pull/8663
- https://github.com/Kong/kong/pull/8914
- https://github.com/Kong/kong/pull/8574

Please merge with Rebase.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -137,7 +137,9 @@
 - Bumping version number (`_format_version`) of declaritive configuration to "3.0" for changes on `route.path`.
   Declaritive configuration with older version are upgraded to "3.0" on the fly.
   [#9078](https://github.com/Kong/kong/pull/9078)
-
+- Removed deprecated `config.functions` from serverless-functions plugin's schema,
+  please use `config.access` phease instead.
```
Comment (chronolaw) [inline CHANGELOG.md:141]: phease => phase ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -353,30 +358,32 @@
...
-  being reloaded. [#8702](https://github.com/Kong/kong/pull/8702)
+  being reloaded. 
+  [#8702](https://github.com/Kong/kong/pull/8702)
 - The private stream API has been rewritten to allow for larger message payloads
   [#8641](https://github.com/Kong/kong/pull/8641)
 - Fixed an issue that the client certificate sent to upstream was not updated when calling PATCH Admin API
   [#8934](https://github.com/Kong/kong/pull/8934)
 - Fixed an issue where the CP and wRPC modules would cause Kong to crash when calling `export_deflated_reconfigure_payload` without a pcall
   [#8668] https://github.com/Kong/kong/pull/8668
+- Moved all `.proto` files to `/usr/local/kong/include` and order by priority.
```
Comment (chronolaw) [inline CHANGELOG.md:380]: order => ordered ?

File: CHANGELOG.md (hunk 1 above)
Comment (mayocream) [reply CHANGELOG.md]: Fixed!

File: CHANGELOG.md (hunk 2 above)
Comment (mayocream) [reply CHANGELOG.md]: Done!

**Review by chronolaw — APPROVED** [review]

**Review by dndx — APPROVED** [review]

**Comment by dndx** [comment]
@mayocream could you resolve the merge conflicts in this PR from earlier merges into `master`?

**Comment by mayocream** [comment]
> @mayocream could you resolve the merge conflicts in this PR from earlier merges into `master`?

Rebased on latest master branch :)

**Pull request merged**